src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -239,26 +251,17 @@ def get(self, request: Request, project: Project) -> Response:
                     break
 
             # Post-processing before exiting scope context
-            found: bool = result["sourceUrl"] is not None
-            scope.set_tag("stacktrace_link.found", found)
-            scope.set_tag("stacktrace_link.source_url", result.get("sourceUrl"))
-            scope.set_tag("stacktrace_link.tried_url", result.get("attemptedUrl"))
             if current_config:
                 result["config"] = current_config["config"]
-                scope.set_tag(
-                    "stacktrace_link.auto_derived",
-                    result["config"]["automaticallyGenerated"] is True,
-                )
-                scope.set_tag("stacktrace_link.empty_root", result["config"]["stackRoot"] == "")
-                if not found:
+                if not result.get("sourceUrl"):
                     result["error"] = current_config["outcome"]["error"]
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
-                    if result["error"] == "stack_root_mismatch":
-                        scope.set_tag("stacktrace_link.error", "stack_root_mismatch")
-                    else:
-                        scope.set_tag("stacktrace_link.error", "file_not_found")
+            try:
+                set_tags(scope, result)
```

Comment:
Setting of tags should not make the API fail, however, we should report to Sentry so we can investigate.